assets/css/style.css

```diff
@@ -222,4 +222,24 @@ input[type="checkbox"] {
 h1 {
     font-family: "Handwriting", serif;
     font-size: 3rem;
-}
\ No newline at end of file
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+}
+
+.simplist-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 15px;
```

Comment:
Change sizing from px to either rem or em. I tried to design everything so it works with whole or half numbers. Round up to 1 em. (1em = 16px, but it's more responsive)